fv3core/fv3core/stencils/mapn_tracer.py

```diff
@@ -8,6 +8,9 @@
 from pace.dsl.dace.orchestrate import computepath_method
 from pace.dsl.stencil import StencilFactory
 from pace.dsl.typing import FloatField
+
+# [DaCe] Import.
+#        Quantity required for type hints
```

Comment:
Remove comment?

Reply:
Good catch

Reply:
```
.tox/test/lib/python3.8/site-packages/fv3core/stencils/mapn_tracer.py:32: in MapNTracer
    tracers: Dict[str, Quantity],
E   NameError: name 'Quantity' is not defined
```
sigh

Reply:
If you want to put `Quantity` in a type hint, the best practice is to import it. It's possible to type hint with quotes without the import, but the import makes it explicitly clear what class you are referring to with the type hint.

It doesn't really matter if you never use the symbol other than for type hints.